### PR TITLE
User/t limay/style transfer skeleton

### PR DESCRIPTION
**Pull request by linnealovespie**
Creating skeleton project for new styletransfer demo.

### CODE REVIEW

File: Samples/StyleTransfer/MainPage.xaml.cs [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
```
Comment (cartazonon) [inline Samples/StyleTransfer/MainPage.xaml.cs:13]: These should be Microsoft.UI.* right?

File: Samples/StyleTransfer/App.xaml.cs [hunk 2]
```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using Windows.ApplicationModel;
+using Windows.ApplicationModel.Activation;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.UI.Xaml;
```
Comment (smk2007) [inline Samples/StyleTransfer/App.xaml.cs:10]: This should be Microsoft.UI.* right?

File: Samples/StyleTransfer/StyleTransfer.csproj [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">x86</Platform>
+    <ProjectGuid>{CF93BFB0-F3B8-4D68-A044-07AA586221F0}</ProjectGuid>
+    <OutputType>AppContainerExe</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>StyleTransfer</RootNamespace>
+    <AssemblyName>StyleTransfer</AssemblyName>
+    <DefaultLanguage>en-US</DefaultLanguage>
+    <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
```
Comment (smk2007) [inline Samples/StyleTransfer/StyleTransfer.csproj:13]: Looks like this project is targeting UAP, it would be nice if this targeted a desktop application?

Comment (linnealovespie) [reply Samples/StyleTransfer/StyleTransfer.csproj:13]: Doesn't UAP cover desktop applications?

Comment (smk2007) [reply Samples/StyleTransfer/StyleTransfer.csproj:13]: unfortunately not :(

File: Samples/StyleTransfer/App.xaml.cs (hunk 2 above)
Comment (smk2007) [reply Samples/StyleTransfer/App.xaml.cs:10]: Yeah... interesting. It seems like the WinUI Microsoft.* is missing MediaElement, and MediaPlayerElement per: https://docs.microsoft.com/en-us/windows/apps/winui/winui3/#preview-1-limitations-and-known-issues.

**Comment by zhangxiang1993** [comment]
> Creating skeleton project for new styletransfer demo.

Please build the sample in azure-pipelines-samples.yml, also [add tests in SampleTests in WAI](https://microsoft.visualstudio.com/WindowsAI/_releaseDefinition?definitionId=8&_a=definition-tasks&environmentId=72)

**Comment by linnealovespie** [comment]
> 
> 
> > Creating skeleton project for new styletransfer demo.
> 
> Please build the sample in azure-pipelines-samples.yml, also [add tests in SampleTests in WAI](https://microsoft.visualstudio.com/WindowsAI/_releaseDefinition?definitionId=8&_a=definition-tasks&environmentId=72)

Will build the sample and add tests once the sample gets closer to being finished.

**Pull request merged**